### PR TITLE
Add support for source maps with optimize=none

### PR DESCRIPTION
**Pull request by chuckdumont**
Allows you to work with individual source files when debugging rather than the aggregated source.

### CODE REVIEW

**Comment by chuckdumont** [comment]
@ddumont please review

File: jaggr-core/src/test/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/IdentitySourceMapGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import junit.framework.Assert;
+
+public class IdentitySourceMapGeneratorTest {
+	private static final List<JSSourceFile> externs = Collections.emptyList();
+
+	static final String source =
```
Comment (ddumontatibm) [inline jaggr-core/src/test/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/IdentitySourceMapGeneratorTest.java]: What's this?

Comment (ddumontatibm) [inline jaggr-core/src/test/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/IdentitySourceMapGeneratorTest.java]: oh just a test... nevermind

**Comment by ddumontatibm** [comment]
lgtm

**Pull request merged**